java/src/main/java/com/adhoc/flight/QueryRunner.java

```diff
@@ -1,18 +1,20 @@
 /*
- * Copyright (C) 2017-2021 Dremio Corporation
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Let's not change the heading to be under ASF since this isn't Apache work. Edit the checkstyle file for this.

Reply:
Fixed